finagle-benchmark/src/main/scala/com/twitter/finagle/loadbalancer/RoundRobin.scala

```diff
@@ -0,0 +1,84 @@
+package com.twitter.finagle.loadbalancer
+
+import com.twitter.finagle.{ClientConnection, NoBrokersAvailableException, Service, ServiceFactory}
+import com.twitter.finagle.service.FailingFactory
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.finagle.util.OnReady
+import com.twitter.util.{Activity, Future, Promise, Time}
+
+object RoundRobinBalancer {
```

Comment:
Sure, I can move it and improve the scaladoc to match what you'd expect from something in finagle-core. I'm surprised you have a serious use case for Round Robin that isn't better served by almost any other choice currently in finagle. Would you also want it exposed easily as Balancers.rr?